### PR TITLE
MM-25651 add ldap idmigrate command

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://mattermost.atlassian.net/browse/MM-25651 & https://github.com/mattermost/mmctl/pull/169

Updated:
- Self-Managed Admin Guide > Administration > mmctl Command Line Tool > mmctl ldap
   - Added ``mmctl ldap idmigrate``

### CODE REVIEW

File: source/administration/mmctl-cli-tool.rst [hunk 1]
```diff
@@ -2982,6 +2982,7 @@ mmctl ldap
 LDAP-related utilities.
 
    Child Commands
+      -  `mmctl ldap idmigrate` - Migrate LDAP IdAttribute to a new value
       -  `mmctl ldap sync`_ - Sync all LDAP users and groups
```
Comment (cwarnermm) [inline source/administration/mmctl-cli-tool.rst:2986]:
```suggestion
      -  `mmctl ldap sync` - Sync all LDAP users and groups
```

File: source/administration/mmctl-cli-tool.rst [hunk 2]
```diff
@@ -2990,6 +2991,42 @@ LDAP-related utilities.
 
    -h, --help   help for ldap
 
+mmctl ldap idmigrate
+~~~~~~~~~~~~~~~~~~~~
+
+**Description**
+
+Migrate the LDAP IdAttribute to a new value. Run this utility then change the IdAttribute to the new value.
```
Comment (cwarnermm) [inline source/administration/mmctl-cli-tool.rst:2999]:
```suggestion
Migrate LDAP ``IdAttribute`` to a new value. Run this utility then change the IdAttribute to the new value.
```

File: source/administration/mmctl-cli-tool.rst [hunk 3]
```diff
@@ -2982,14 +2982,51 @@ mmctl ldap
 LDAP-related utilities.
 
    Child Commands
-      -  `mmctl ldap sync`_ - Sync all LDAP users and groups
+      -  `mmctl ldap idmigrate` - Migrate LDAP IdAttribute to a new value
```
Comment (cwarnermm) [inline source/administration/mmctl-cli-tool.rst:2985]:
```suggestion
      -  `mmctl ldap idmigrate`_ - Migrate LDAP IdAttribute to a new value
```

File: source/administration/mmctl-cli-tool.rst [hunk 4]
```diff
@@ -2982,14 +2982,51 @@ mmctl ldap
...
+      -  `mmctl ldap sync` - Sync all LDAP users and groups
```
Comment (cwarnermm) [inline source/administration/mmctl-cli-tool.rst:2986]:
```suggestion
      -  `mmctl ldap sync`_ - Sync all LDAP users and groups
```

File: source/administration/mmctl-cli-tool.rst [hunk 5]
```diff
@@ -2990,6 +2991,42 @@ LDAP-related utilities.
...
+
+**Description**
+
+Migrate LDAP ``IdAttribute`` to a new value. Run this utility then change the IdAttribute to the new value.
+
+**Format**
+
+.. code-block:: sh
+
+   mmctl ldap idmigrate [flags]
```
Comment (mgdelacroix) [inline source/administration/mmctl-cli-tool.rst:3005]: The usage should include the `objectGUID` as a positional argument. This should be added to the `mmctl` code as well.

```suggestion
   mmctl ldap idmigrate <objectGUID> [flags]
```

Comment (mgdelacroix) [reply source/administration/mmctl-cli-tool.rst:3005]: I've just sent a PR with this change to `mmctl` here: https://github.com/mattermost/mmctl/pull/356

**Review by wiersgallak — APPROVED** [review]
Good from my side with updates added requested by Miguel.

**Comment by cwarnermm** [comment]
Thanks so much for the update, @mgdelacroix!

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**